src/utils/prim-lexing.fs

```diff
@@ -167,17 +166,17 @@ namespace Internal.Utilities.Text.Lexing
             LexBuffer<'Char>.FromArrayNoCopy buffer
 
         // Important: This method takes ownership of the array
-        static member FromChars (arr:char[]) = LexBuffer.FromArrayNoCopy arr 
+        static member FromChars (arr:char[]) = LexBuffer.FromArrayNoCopy (arr |> Array.map uint16)
```

Comment:
We are now making a new array. Before, the intent was not to do so.

Reply:
Yes, looks like this one can be improved, as it's always used with a new array, like that: `LexBuffer<char>.FromChars (s.ToCharArray())`
I'll fix it.